kubectl.ts

```diff
@@ -28,11 +28,9 @@ async function run() {
         tl.debug("DEBUG:  " + kubectlbinary + " apply -f " + yamlfile + " --kubeconfig ./config")
 
         let kubectl: ToolRunner = tl.tool(kubectlbinary).arg('apply').arg('-f').arg(yamlfile).arg('--kubeconfig').arg('./config');
-        let result =  kubectl.execSync();
-        tl.debug("STDOUT: " + result.stdout);
-        tl.debug("STDERR: " + result.stdout);
+        await kubectl.exec();
```

Comment:
default is to throw when exit code is not zero, so i didnt bother capturing the return code. exec only returns return code and doesnt return an object stdout/stderr like execSync.

if i recall correctly, by default stdout/stderr are written to the underlying process.stdout/stderr. if that is not desired, then there is an option to tool runner to make it silent.